### DISCUSSION TITLE
CVE-2024-21742: CWE-20 Improper Input Validation apache-mime4j-core@0.8.9

### DISCUSSION THREAD

**Discussion by AtulShu** [keycloak/keycloak#29502] [Q&A]
We have CVE-2024-21742 that is coming from apache-mime4j-core@0.8.9. Is there any plan to release a new version to fix this vulnerability. This is the dependency path. 
<img width="169" alt="image" src="https://github.com/keycloak/keycloak/assets/35775223/684e55f5-f3ca-413b-98ca-371c0cb02bcf">

I can see we have one open ticket also https://github.com/keycloak/keycloak/issues/27372 for the same that was opened way back in february. But there is no update on that also.

**Comment by jonkoops** [comment]
Feel free to submit a PR to fix this issue @AtulShu
